src/Deals.php

```diff
@@ -83,6 +95,7 @@ public function getBySearch($search, $limit = false, $offset = 0)
 	public function getBaseQuery($limit = false, $offset = 0)
 	{
 		$query = DB::table('deals')
+					->join('merchants','deals.nMerchantID','=','merchants.nMerchantID')
```

Comment:
There should be spaces after these commas to match the style of the rest of the project.